stl/inc/mdspan

```diff
@@ -289,15 +295,21 @@ public:
     using rank_type    = typename _Extents::rank_type;
     using layout_type  = layout_left;
 
+    static_assert(
+        _Is_extents<_Extents>, "Extents must be a specialization of extents (N4928 [mdspan.layout.left.overview]/2).");

```

Comment:
Suggestion: Although we often omit `std::` in messages because it's clear we're the Standard Library, for things that resemble ordinary words, qualifying them can be helpful to avoid confusion. `std::function` and `std::string` are the classic examples. Here, it seems that "Extents must be a specialization of std::extents` would be clearer.